inc/maiko/platform.h

```diff
@@ -145,6 +145,10 @@
 #  else
 #    error "Unknown byte order"
 #  endif
+#elif __BIG_ENDIAN__ == 1
```

Comment:
Could you add a comment here that this is for old gcc?

Reply:
Sure can.  I don't know in which version of GNU C or Clang they introduced `__BYTE_ORDER__` but I do know when it _didn't_ have it.